crates/solidity/testing/snapshots/cst_output/Expression/index_missing/generated/0.4.11.yml

```diff
@@ -3,17 +3,12 @@
 Source: >
```

Comment:
I wonder why did `IndexAccessOperator` rule change? the old grammar prevented an empty index expression, while the new one allows it, which is a bug.

Reply:
Discussed f2f. This is needed. Let's remove the `index_missing` test then.